cmd/cluster/azure/create.go

```diff
@@ -83,12 +84,27 @@ func applyPlatformSpecificsValues(ctx context.Context, exampleOptions *apifixtur
 			InfraID:         infraID,
 			CredentialsFile: opts.AzurePlatform.CredentialsFile,
 			BaseDomain:      opts.BaseDomain,
+			PullSecret:      opts.PullSecretFile,
+			ReleaseImage:    opts.ReleaseImage,
 		}).Run(ctx, opts.Log)
 		if err != nil {
 			return fmt.Errorf("failed to create infra: %w", err)
 		}
 	}
 
+	var instanceType string
+	if opts.AzurePlatform.InstanceType != "" {
+		instanceType = opts.AzurePlatform.InstanceType
+	} else {
+		// Aligning with Azure IPI instance type defaults
```

Comment:
is there a link to this ipi defaults?

Reply:
I had the wrong x86 one in there - should be Standard_D4s_v3 instead of v4. I'll fix that. For x86, the documentation is here - https://access.redhat.com/documentation/en-us/openshift_container_platform/4.11/html/installing/installing-on-azure. I don't see anything for arm in there but I selected the equivalent arm instance type based on vCPU, Memory, etc based on the specs for Standard_D4s_v3.